math/fibonacci_fast.cpp

```diff
@@ -24,23 +24,23 @@
 const uint64_t MAX = 93;
 
 /** Array of computed fibonacci numbers */
-uint64_t f[MAX] = {0};
+uint64_t numbers[MAX] = {0};
```

Comment:
better to replace this C-style program with `std::vector` and `push_back()` function. 
Or better still, for this particular application, using 
`static uint64_t f0` and `static uint64_t f1` within the `fib()` function would be enough. no need for an array at all.